install_config/install/docker_registry.adoc

```diff
@@ -686,9 +686,16 @@ Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 
 You can override the integrated registry's default configuration, found by
 default at *_/config.yml_* in a running registry's container, with your own
-custom configuration. See the upstream registry documentation's
-https://docs.docker.com/registry/configuration/[Registry Configuration
-Reference] for the full list of available options.
+custom configuration. See
+link:#registry-configuration-reference[registry configuration reference] for more
+information.
+
+[TIP]
+Every configuration option in this file may also be overridden using environment
```

Comment:
This is not true. Middleware options cannot be overriden by environment variables.